response_operations_ui/views/collection_exercise.py

```diff
@@ -649,16 +642,16 @@ def create_collection_exercise(survey_ref, short_name):
                 survey_ref=survey_ref,
                 survey_id=survey_id,
                 survey_name=survey_name,
-                survey_eq_version=survey_eq_version,
             )
 
     logger.info(
         "Creating collection exercise for survey",
         survey=short_name,
         survey_ref=survey_ref,
-        survey_eq_version=survey_eq_version,
     )
 
+    survey_eq_version = "v3" if survey_details["surveyMode"] == "EQ" else ""
```

Comment:
This should only be temporary, but I can't remove this until the column has been removed from the db, which can't be done at present